node/src/cli.rs

```diff
@@ -15,4 +15,20 @@ pub struct Cli {
     /// specified multiple times
     #[arg(long)]
     pub addnode: Option<Vec<String>>,
+
+    /// Connect to this bitcoin RPC node
+    #[arg(long, default_value = "0.0.0.0:38332")]
+    pub rpc_url: String,
+
+    /// Use this username for bitcoin RPC
+    #[arg(long, default_value = "braidpooldev")]
+    pub rpc_user: String,
+
+    /// Use this password for bitcoin RPC
+    #[arg(long, default_value = "braidpooldev")]
```

Comment:
Lets take out the default username and password. Instead lets make these required. So we do not assume or proliferate default passwords that people use everywhere and thus get messed with.